src/utils/enums/modes.ts

```diff
@@ -0,0 +1,19 @@
+export enum OutMode {
+    out = "out",
+    bounce = "bounce",
+    bounceVertical = "bounce-vertical",
+    bounceHorizontal = "bounce-horizontal",
+}
+
+export enum HoverMode {
+    grab = "grab",
+    bubble = "bubble",
+    repulse = "repulse",
+}
+
+export enum ClickMode {
+    push = "push",
+    remove = "remove",
+    bubble = "bubble",
+    repulse = "repulse",
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [file should end with a newline](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5012291)